lib/barbeque/config.rb

```diff
@@ -0,0 +1,49 @@
+require 'erb'
+require 'yaml'
+
+module Barbeque
+  class Config
+    attr_accessor :exception_handler, :runner, :runner_options
+
+    def initialize(options = {})
+      options.each do |key, value|
+        if respond_to?("#{key}=")
+          public_send("#{key}=", value)
+        else
+          raise KeyError.new("Unexpected option '#{key}' was specified.")
+        end
+      end
+      init_runner_options
+    end
+
+    private
+
+    def init_runner_options
+      self.runner_options ||= {}
+      self.runner_options = Hash.new.tap do |symbolized|
```

Comment:
???? It was misbelief. Never mind. https://github.com/cookpad/barbeque/pull/11/commits/400ba6fa701b1349156b18b14603fd64b32b4d8f